app/middleWare/auth.js

```diff
@@ -0,0 +1,32 @@
+const jwt = require("jsonwebtoken");
+const config = require("config");
+
+module.exports = (req, res, next) => {
+  // Get the token from header;
+  const token = req.header("x-auth-token");
```

Comment:
Why is the header name `x-auth-token` and not `Authorization`? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

Reply:
Thanks for the link and will change the header name to "Authorization".